src/main/java/org/jabref/gui/collab/DatabaseChangeMonitor.java

```diff
@@ -34,9 +30,6 @@ public DatabaseChangeMonitor(BibDatabaseContext database, FileUpdateMonitor file
         this.database.getDatabasePath().ifPresent(path -> {
             try {
                 fileMonitor.addListenerForFile(path, this);
-                referenceFile = Files.createTempFile("jabref", ".bib");
```

Comment:
I like that removal - even though I think, one still needs that reference somehow. Maybe only for the approach sketched at https://github.com/JabRef/jabref/pull/5669.